### ISSUE TITLE
关于obtain_proposals与results_totals

### ISSUE THREAD

**Issue by snowindog** [nullmax-vision/QAF2D#6]
直接在nuscenes_dataset.py  生成
                      results_plane=results_totals,在obtain_proposals里直接用这个生成initial_proposals 是否达成了文章图二所示的流程，

2D检测框都没利用
        if not requires_grad:
            self.eval()
            with torch.no_grad():
                outs = self.pts_bbox_head(
                    img_metas, init_proposals=init_proposals, **data)
            self.train()
        else:
            outs_roi = self.forward_roi_head(**data)
            outs = self.pts_bbox_head(
                img_metas, init_proposals=init_proposals, **data)

        if return_losses:
            loss_inputs = [gt_bboxes_3d, gt_labels_3d, outs]
            losses = self.pts_bbox_head.loss(*loss_inputs)
            if self.with_img_roi_head:
                loss2d_inputs = [gt_bboxes, gt_labels,
                                 centers2d, outs_roi, depths, img_metas]
                losses2d = self.img_roi_head.loss(*loss2d_inputs)
                losses.update(losses2d)

            return losses
        else:
            return None
